### ISSUE TITLE
[taildrop] unable to send files when running in userspace-networking mode

### ISSUE THREAD

**Issue by maisem** [tailscale/tailscale#3269]
### What is the issue?

When running in userspace-networking mode, I am unable to send files over Taildrop.

```
$ tailscale file cp hello maisems-iphone-1:
Put "http://100.118.149.75:43959/v0/put/hello": dial tcp 100.118.149.75:43959: connect: no route to host
```

### OS

Linux

### Tailscale version

1.17.186

**Comment by DentonGentry** [comment]
I would have sworn that there was an existing bug for this, but I cannot find it either.

**Comment by maisem** [comment]
I think this is fixed as of 8775c646be461e098b50e16b6c189b576669f2b5 and #2179
